### PR TITLE
Add Kiwix.app v0.9.

### PR DESCRIPTION
**Pull request by adwylie**
Added a cask for Kiwix (current stable version).

### CODE REVIEW

**Comment by adityadalal924** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git)

**Comment by awylie** [comment]
Commits have been squashed, thanks!

File: Casks/kiwix.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'kiwix' do
+  version '0.9'
+  sha256 '98e677aee3c106c6ec5b16791d3be7c22274b3b32cc44a9cce1eeca6275b7bcc'
+
```
Comment (adityadalal924) [inline Casks/kiwix.rb]: One more thing I noticed, needs the comment
`# sourceforge.net is the official download host per the vendor homepage` here (above the `url` stanza)

Thanks!

**Comment by awylie** [comment]
Sure.  Why though?, this isn't mentioned in the [contributing document](https://github.com/caskroom/homebrew-cask/blob/master/CONTRIBUTING.md) (how was I supposed to know to do that?).

**Comment by adityadalal924** [comment]
Sorry, it's documented in the language reference [here](https://github.com/caskroom/homebrew-cask/blob/master/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)

Thanks for your contribution!

**Comment by awylie** [comment]
Should be all done, or is there anything else I have to do?

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 9701df0e120f2bbd0d73e7c3457c44540f7905b8 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**